src/modules/RF69/RF69.cpp

```diff
@@ -369,6 +369,31 @@ int16_t RF69::readData(uint8_t* data, size_t len) {
   return(ERR_NONE);
 }
 
+int16_t RF69::setOOK(bool enableOOK) {
+    // set OOK and if successful, save the new setting
+    int16_t state = ERR_NONE;
+    if(enableOOK) {
+        state = _mod->SPIsetRegValue(RF69_REG_DATA_MODUL, RF69_OOK, 4, 3, 5);
+    } else {
+        state = _mod->SPIsetRegValue(RF69_REG_DATA_MODUL, RF69_FSK, 4, 3, 5);
+    }
+    if(state == ERR_NONE) {
+        _ook = enableOOK;
+    }
+
+    return(state);
+}
+
+int16_t RF69::setOokThresholdType(uint8_t type) {
+  if(type != RF69_OOK_THRESH_FIXED && type != RF69_OOK_THRESH_PEAK && RF69_OOK_THRESH_AVERAGE)
```

Comment:
This doesn't seem correct, I think the last part should be `&& type != RF69_OOK_THRESH_AVERAGE` - please use parentheses to make the intent clear.

Reply:
> Hey, thanks for the PR - I left a couple comments, nothing major, mostly just style. Is there a specific reason for these additions (e.g. compatibility with existing hardware)?

Hi! My project mostly does OOK reception and your great library has a robust core but has simply put OOK on the side !

Reply:
good catch!!!!